### PR TITLE
chore(workflow-rc2): releasing workflow-rc2

### PR DESCRIPTION
**Pull request by krancour**

### CODE REVIEW

File: src/managing-workflow/upgrading-workflow.md [hunk 1]
```diff
@@ -67,27 +67,27 @@ $ kubectl --namespace=deis get secret database-creds -o yaml > ~/active-deis-dat
...
-	$(helmc home)/workspace/charts/workflow-rc1-manifests/deis-database-secret-creds.yaml
+	$(helmc home)/workspace/charts/workflow-rc2-manifests/deis-database-secret-creds.yaml
 
 # copy your active builder ssh keys into the helmc workspace
 $ cp ~/active-deis-builder-secret-ssh-private-keys.yaml \
-	$(helmc home)/workspace/charts/workflow-rc1/manifests/deis-builder-secret-ssh-private-keys.yaml
+	$(helmc home)/workspace/charts/workflow-rc2/manifests/deis-builder-secret-ssh-private-keys.yaml
 
 # uninstall workflow
 $ helmc uninstall workflow-beta4 -n deis
```
Comment (vdice) [inline src/managing-workflow/upgrading-workflow.md]: though `beta4` to `rc2` as an upgrade path should work (right?), we usually change this chart to the previous release, which would be `workflow-rc1` in this case.

Comment (krancour) [inline src/managing-workflow/upgrading-workflow.md]: Let me double-check that.

Comment (krancour) [inline src/managing-workflow/upgrading-workflow.md]: Ya... there are some beta4 refs that should have been updated. Finding and fixing.

Comment (krancour) [inline src/managing-workflow/upgrading-workflow.md]: Fixed.

**Pull request merged**